internal/test/scorecard/clients.go

```diff
@@ -89,6 +89,15 @@ func (c *OperatorCRDClient) Cryostats(namespace string) *CryostatClient {
 	}
 }
 
+// Cryostats returns a ClusterCryostatClient configured to a specific namespace
+func (c *OperatorCRDClient) ClusterCryostats(namespace string) *CryostatClient {
+	return &CryostatClient{
+		restClient: c.client,
+		namespace:  namespace,
+		resource:   "clustercryostats",
+	}
+}
+
```

Comment:
Since `ClusterCryostat` is non-namespaced, would it make sense to not specify namespace?

```go
// Cryostats returns a ClusterCryostatClient
func (c *OperatorCRDClient) ClusterCryostats() *CryostatClient {
	return &CryostatClient{
		restClient: c.client,
		resource:   "clustercryostats",
	}
}
```